tests/unit-tests/wpml/test-class-sensei-wpml.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace SenseiTest\WPML;
+
+use \Sensei_WPML;
+
+/**
+* Class Sensei_WPML_Test.
+*
+* @covers \Sensei_WPML
+*/
+class Sensei_WPML_Test extends \WP_UnitTestCase {
+	public function testSetLanguageDetailsWhenLessonCreated_WhenCalled_AppliesWpmlElementLangugeCodeFilter() {
+		/* Arrange. */
+		$wpml = new Sensei_WPML();
+
+		$filter_applied = false;
+		$filter_funtction = function( $language_code, $element_data ) use ( &$filter_applied ) {
```

Comment:
Another small typo here - `filter_funtction` -> `filter_function`

Typos also apply to the other test case. 🙂